build-python-package/action.yml

```
@@ -46,4 +50,4 @@ runs:
       if: inputs.check-dependencies == 'true'
       # ^ string comparison needed due to https://github.com/actions/runner/issues/2238
       shell: bash -leo pipefail {0}
-      run: python3 -m pip install -e .[all] --dry-run --ignore-installed
+      run: python3 -m pip install -e .[${{ inputs.optional-deps }}] --dry-run --ignore-installed
```

It's not possible to install just the core dependencies like this, using `optional-deps = ''`

```
$ pip install -e .[]
ERROR: .[] is not a valid editable requirement. It should be ...
```


```suggestion
      run: python3 -m pip install -e .${{ inputs.optional-deps }} --dry-run --ignore-installed
```